jax/_src/cudnn/fused_attention_stablehlo.py

```diff
@@ -36,6 +37,17 @@
 DType = jnp.dtype
 PRNGKey = jnp.ndarray
 
+class AttentionLayout(Enum):
+  BTNH = 0
+  BNTH = 1
+
+def _normalize_layout(layout_str):
```

Comment:
How about just `layout`? You can use a type annotation to document the type:

```
def _normalize_layout(layout: str) -> AttentionLayout:
  ...
```

Reply:
Yes, this is better. Fixed.